libqtile/utils.py

```diff
@@ -266,10 +266,10 @@ def lazify_imports(
     """
     __all__ = tuple(registry.keys())
 
-    def __dir__() -> tuple[str, ...]:
+    def __dir__() -> tuple[str, ...]:  # noqa: N807
```

Comment:
Why is ruff adding a `noqa` comment when we didn't need it before?

Reply:
also i added that one manually

Reply:
seems to be a mistake from ruff

https://github.com/PyCQA/pep8-naming/blob/f0edf20bd88cbd0950240ebeeea4fe6f4d90a094/CHANGELOG.rst?plain=1#L136-L137

Reply:
Yes - it's a valid use of those names.

Reply:
i did some digging and
- `pep8-naming`s N807 just ignores anything called `__dir__` and `__getattr__`
- `ruff`s N807 only ignores top level `__dir__` and `__getattr__`